sources/zsh/aliases

```diff
@@ -13,4 +13,5 @@ alias ipa='ip --brief --color a'
 alias urlencode='python -c "import sys, urllib as ul; print ul.quote_plus(sys.argv[1])"'
 alias urldecode='python -c "import sys, urllib as ul; print ul.unquote_plus(sys.argv[1])"'
 alias sed-empty-line='sed /^$/d'
-alias http-put-server='python3 /opt/resources/linux/http-put-server.py --bind 0.0.0.0'
\ No newline at end of file
+alias http-put-server='python3 /opt/resources/linux/http-put-server.py --bind 0.0.0.0'
+alias www="python3 -m http.server 8000"
```

Comment:
Concerning the `www` alias, it would be better not to specify the port here to allow the user to change it if needed while using the alias, for example: `www 8080`

Reply:
Good idea ! 👍